Models/Question.cs

```diff
@@ -14,5 +14,13 @@ public class Question
         [ForeignKey("QuizId")]
         public int QuizId { get; set; }
         public string? UserId { get; set; } = null;
+        public Difficulty Difficulty { get; set; } = 0;
+    }
+
+    public enum Difficulty
+    {
+        EASY = 1,
+        MEDIUM,
+        HARD
     }
```

Comment:
Create an enum value like NOT_GIVEN / NOT_ASSIGNED / NOT_AVAILABLE or something similar instead of setting to 0